core/domain/takeout_service.py

```diff
@@ -44,8 +44,8 @@ def get_models_which_should_be_exported():
     """
     return [model_class for model_class in
             models.Registry.get_all_storage_model_classes()
-            if base_models.EXPORT_POLICY.EXPORTED in
-            model_class.get_export_policy().values()]
+            if base_models.EXPORT_METHOD.NOT_EXPORTED !=
```

Comment:
This reads slightly oddly, perhaps reverse them (if get_export_method() != constant)

Reply:
Done